load_tests/load_test.py

```diff
@@ -6,36 +6,34 @@
 import subprocess
 from datetime import datetime, timezone
 
-PLATFORM = os.environ['PLATFORM']
+PLATFORM = os.environ['PLATFORM'].lower()
```

Comment:
how we decide when the os environment is upper case and when is lower case? could this be unified? Same comment for environment under this line.

Reply:
it is set upper case in our pipeline environment just because by convention, environment variables are written in UPPERCASE, but it is used as lower case in our script so we convert it here. Hmm, I am fine to change it but want to discuss with the team first

Reply:
I think this makes sense

Reply:
Doesn't this make the value of the env var lower case, Not the env var itself? Also I was saying I agree with Zhonghui, env vars are always upper case I think.

Reply:
yeah, not the env var itself